src/main/java/org/killbill/billing/plugin/vertex/VertexTaxCalculator.java

```diff
@@ -285,6 +287,15 @@ private Collection<InvoiceItem> toInvoiceItems(final UUID invoiceId,
         }
     }
 
+    private static String getDescription(final TaxesType transactionLineDetailModel) {
+        final Jurisdiction jurisdiction = transactionLineDetailModel.getJurisdiction();
+        if (jurisdiction != null) {
+            return String.format("%s %s TAX", jurisdiction.getValue(), jurisdiction.getJurisdictionType());
+        }
+
+        return "TAX";
```

Comment:
Curious why do you change MoreObjects.firstNonNull(transactionLineDetailModel.getTaxCode(), MoreObjects.firstNonNull(transactionLineDetailModel.getVertexTaxCode(), "Tax")) to just "TAX"?